src/components/modular-content/responsive-image/responsive-image.njk

```diff
@@ -8,21 +8,32 @@
         class="responsive-image__fixed-ratio"
         style="--image-width: {{ blockData.imageWidth or 100}}%"
       >
-        <span
-          class="responsive-image__spacer"
-          style="padding-top: {{ 100.0 / image.responsiveImage.aspectRatio }}%;
-              background-image: url({{ image.responsiveImage.base64 }});"
-        ></span>
-        <picture class="responsive-image__picture">
-          <source type="image/webp" srcset="{{ image.url | imageUrl({ w: 776, fm: 'webp' }) }}">
+        {% if image.format === 'svg' %}
           <img
-            alt="{{ image.alt }}"
-            src="{{ image.url | imageUrl({ w: 776 }) }}"
-            loading="lazy"
-            class="responsive-image__figure-image"
-            style="width:100%; height:100%;"
-          />
-        </picture>
+              alt="{{ image.alt }}"
+              src="{{ image.url }}"
+              loading="lazy"
+              width="{{ image.width }}"
+              height="{{ image.height }}"
+              class="responsive-image__figure-image"
+            />
+        {% else %}
+          <span
+            class="responsive-image__spacer"
+            style="padding-top: {{ 100.0 / image.responsiveImage.aspectRatio }}%;
```

Comment:
This already existed, but just out of curiosity, why `100.0`?

Reply:
example of image with `width: 980 / height:471`
dato aspect ratio is: `2.08067940552017`
to calc the correct padding top value: 100/aspect ratio